.github/workflows/build_native_linux_packages.yml

```
@@ -43,9 +49,13 @@ on:
           - deb
         default: "rpm"
       package_suffix:
-        description: The suffix to be added to package name(build_no or master or rc or combiantion).
-        type: false
-        default: "test"
+        description: The suffix to be added to package name (asan, static or rpath).
+        type: string
+        required: false
+      release_type:
+        description: The type of build version to produce (nightly, prerelease or dev).
+        required: false
+        type: string
```

Default to "dev", as is done here: https://github.com/ROCm/TheRock/blob/efed3c3b10a5cce8578f58f8eb288582c26d18c4/.github/workflows/release_portable_linux_packages.yml#L36-L39

See https://github.com/ROCm/TheRock/blob/main/docs/development/style_guide.md#use-safe-defaults-for-inputs


```suggestion
      release_type:
        description: The type of build version to produce (nightly, prerelease or dev).
        type: string
        default: "dev"
```